pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
@yegor256 Ok, turns out `@Loggable` is the cause of the false positive, so I just removed it; see ff93dec1c05abad4876e14c0d8a15a12935bf765